kategory/src/main/kotlin/kategory/instances/WriterTInstances.kt

```diff
@@ -27,6 +27,8 @@ interface WriterTInstances<F, W> :
 
 }
 
+interface WriterTMonadWriter<F, W> : MonadWriter<WriterTKindPartial<F, W>, W>
```

Comment:
Since this does not imposes new constrains I think it's fine to implement inline to be consistent how Functor, Applicative and Monad are implemented and there would be no need for the extra interface `WriterTMonadWriter`